ipatests/test_integration/test_automount_locations.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+This module provides tests for the automount location feature.
+"""
+
+from __future__ import absolute_import
+
+import time
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+# give some time for units to stabilize
+# otherwise we get transient errors
+WAIT_AFTER_INSTALL = 5
+WAIT_AFTER_UNINSTALL = WAIT_AFTER_INSTALL
+
+
+class TestAutomountInstallUninstall(IntegrationTest):
+    """
+    Test if vault feature behaves as expected, when KRA is installed or not
```

Comment:
The doc string doesn't match. Did you copy the vault test by any chance?

Reply:
Yes obviously and I missed that. Will fix.